### PR TITLE
Add Appointment Management Feature in User Guide

### LINKED ISSUE #219: Add appointment management feature in UserGuide's feature list
Currently appointment management feature is not in UserGuide's featurelist parameter details and command summary. This issue will fix it.

### PR DESCRIPTION
**Pull request by PresleyChew**
fixes #219 

Added the follow into feature list

- `addAppt`
- `delAppt`

Added the following into parameter details

- `TIME`
- `APPOINTMENTNAME`
- `DATE` (Combined with `DOB`)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -384,9 +385,39 @@ to `91234567` and `johndoe@example.com` respectively.
...
+Format: `addAppt APPOINTMENTNAME i/NRIC @d/DATE @t/TIME`
+
+{: .alert .alert-info}
+> :information_source: **Note:**
+>
+> * Adds Appointment Time and Appointment Name to the patient with the specified `NRIC` in MediBase3.
+> * Appointment time intervals cannot overlap with other existing appointment time intervals.
+> * You can schedule an appointment to a patient even if they are not being currently displayed in the Patient List Panel but doing so will refresh the panel to display all patients after their appointment has been added.
+> * Refer to the [Parameter Details](#parameter-details) section for more information on the purpose and constraints of each parameter.
+
```
[the remark this answers is not in the document]
Comment (yong-zhuo) [reply docs/UserGuide.md:403]: Might also be good to include in the note box or a seperate tip box that adding a new appointment will update the Appointment List Panel with the new appointment details and that the Appointment List is displayed in ascending order

File: docs/UserGuide.md [hunk 2]
```diff
@@ -261,19 +261,20 @@ the constraints of each parameter when used in a command.
...
+| `NRIC`            | Singapore National Registration Identity Card (NRIC) number of the patient. It is unique for all patients. | - Case-insensitive. <br> - Should not be blank. <br> - Should start with a letter (S, F, G or M), followed by 7 digits, and end with a letter.                                                                                                                                       | :white_check_mark: `S1234567A` <br> :white_check_mark: `t1234567b` <br> :x: `1234567A`                                             |
+| `DOB/DATE`        | Date of birth (DOB) of the patient.                                                                        | - Should be in the format `YYYY-MM-DD`. <br> - Should not be blank. <br> - Cannot be a date in the future.                                                                                                                                                                           | :white_check_mark: `2002-12-12` <br> :x: `2002/11/32`                                                                              |
```
[the remark this answers is not in the document]
Comment (PresleyChew) [reply docs/UserGuide.md:268]: Amended

File: docs/UserGuide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (PresleyChew) [reply docs/UserGuide.md:403]: Amended for both

File: docs/UserGuide.md [hunk 3]
```diff
@@ -384,9 +385,53 @@ to `91234567` and `johndoe@example.com` respectively.
...
+> :bulb: **Tip:**
+>
+> * Adding a new appointment will update the Appointment List Panel on the right with the new appointment details.
+> * Appointment List Panel is displayed in chronological order.
+
+[Back to Table of Contents](#table-of-contents)
+
+#### Deleting Appointments : `delAppt`
+
+Allow doctors to delete an appointment with a patient.
```
Comment (hweinian) [inline docs/UserGuide.md:417]: Just a nit: I think it should not be "with a patient" as it sound like you are deleting the patient as well.

**Review by hweinian — CHANGES_REQUESTED** [review]

**Comment by PresleyChew** [comment]
fixes #236 , fixes #238

File: docs/UserGuide.md (hunk 3 above)
Comment (PresleyChew) [reply docs/UserGuide.md]: Amended!

**Pull request merged**